src/coreclr/src/vm/interpreter.h

```diff
@@ -13,6 +13,13 @@
 #include "stack.h"
 #include "crst.h"
 #include "callhelpers.h"
+#include "codeversion.h"
+
+#ifdef PAL_STDCPP_COMPAT
```

Comment:
You do not need this. `PAL_STDCPP_COMPAT` is never defined when building vm.

Reply:
Done in commit https://github.com/dotnet/runtime/pull/37825/commits/74d235a0db1d38ba6ca1345dba11b76be519798a.